Makefile

```diff
@@ -28,15 +28,15 @@ envinit:
 	gomobile init
 # This must be run in $GOPATH/src/github.com/digitalbitbox/bitbox-wallet-app
 osx-init:
-	/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 	brew install yarn
 	brew install go
 	brew install qt
 	export PATH="/usr/local/opt/qt/bin:$PATH"
 	export LDFLAGS="-L/usr/local/opt/qt/lib"
 	export CPPFLAGS="-I/usr/local/opt/qt/include"
 	export GOPATH=~/go/
-	export PATH=$PATH:~/go/bin
+	export PATH=${PATH}:~/go/bin
```

Comment:
This should really be `$(PATH):$(GOPATH)` to make it consistent because in `envinit` target above is using `$(GOPATH)/bin` anyway.

But it isn't nice to overwrite user-set GOPATH. I would've expected it to be defined somewhere at the top of the makefile. For example:

```
$ cat Makefile
GOPATH ?= /default/gopath

dummy:
	@echo $(GOPATH)
```

Then, it works as expected:

```
$ unset GOPATH
$ make
/default/gopath
$ export GOPATH=/let/me/set/my/gopath
$ make
/let/me/set/my/gopath
$ make GOPATH=/another/one
/another/one
$ GOPATH=/another/one make
/another/one
```

Reply:
More about env variables in make: https://www.gnu.org/software/make/manual/html_node/Environment.html